src/test/java/org/tarantool/ClientReconnectIT.java

```diff
@@ -394,9 +407,19 @@ public void execute() throws Throwable {
          * Verify we don't exceed a file descriptor limit. If we exceed it, a
          * client will not able to connect to tarantool.
          */
-        TarantoolClient client = makeClient();
+        TarantoolClient client = makeTestClient(makeDefaultClientConfig(), RESTART_TIMEOUT);
         client.syncOps().ping();
         client.close();
     }
 
+    private TestSocketChannelProvider makeLingerProvider() {
```

Comment:
Here we disable lingering, so it is better to name it like `makeZeroLingerProvider`.